mlearning/tflite-micro/CMakeLists.txt

```diff
@@ -142,14 +135,20 @@ if(CONFIG_TFLITEMICRO)
   # Include Directory
   # ############################################################################
 
+  # cmake-format: off
```

Comment:
ditto

Reply:
cmake-format cannot correctly format the following include path

Reply:
but it.work before?